packages/teams-js/src/public/meeting.ts

```diff
@@ -518,4 +518,23 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
     registerHandler('meeting.meetingReactionReceived', handler);
   }
+  /**
+   * nested namespace for Meeting Service
+   *
+   * @beta
+   */
+  export namespace appShareButton {
+    /**
+     * Allows an app to hide/show app share button in a meeting
+     * @param visible is a boolean flag to set show or hide app share button
+     *
+     * @beta
+     */
+    export function setVisibility(visible: boolean): Promise<void> {
```

Comment:
Does this function actually wait to resolve the promise until the button has been reported as having its visibility changed by the host? I assume not, in which case there's probably no reason to make this return a Promise. The function is "fire and forget", so you can just send a message to the host and keep going.